From 497893dfe05d63c24624c3743fc162e2c759f822 Mon Sep 17 00:00:00 2001
From: "Dr. David von Oheimb" <David.von.Oheimb@siemens.com>
Date: Sun, 24 Jul 2022 22:44:13 +0200
Subject: [PATCH 078/209] 80-test_cmp_http.t: fix adaption of plan on
 'certstatus' aspect of Mock server

Reviewed-by: Hugo Landau <hlandau@openssl.org>
Reviewed-by: Tomas Mraz <tomas@openssl.org>
Reviewed-by: David von Oheimb <david.von.oheimb@siemens.com>
(Merged from https://github.com/openssl/openssl/pull/18858)

(cherry picked from commit 7a86cb69830bf0a8845cd17e606f612109c7564b)
---
 test/recipes/80-test_cmp_http.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/recipes/80-test_cmp_http.t b/test/recipes/80-test_cmp_http.t
index beb3b7a136..26ba349d64 100644
--- a/test/recipes/80-test_cmp_http.t
+++ b/test/recipes/80-test_cmp_http.t
@@ -171,7 +171,7 @@ sub test_cmp_http_aspect {
 
 indir data_dir() => sub {
     plan tests => 1 + @server_configurations * @all_aspects
-        + (grep(/^Mock$/, @server_configurations)
+        - (grep(/^Mock$/, @server_configurations)
            && grep(/^certstatus$/, @all_aspects));
 
     foreach my $server_name (@server_configurations) {
-- 
2.34.1

